.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+---
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: pip
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
This can get really quite noisy, especially if you don't ignore patch versions. At 2i2c our strategy is to run dependabot monthly and ignore patch versions to reduce the number of pings for review we get.

Reply:
Good idea. I've updated the configuration.

I have another repo with weekly alerts so I'll have a chance to see what the difference is like 🙂.